dot/network/block_announce.go

```diff
@@ -20,21 +20,80 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	"math/big"
 
 	"github.com/ChainSafe/gossamer/lib/common"
 	"github.com/ChainSafe/gossamer/lib/scale"
 
 	"github.com/libp2p/go-libp2p-core/peer"
 )
 
+var (
+	_ NotificationsMessage = &BlockAnnounceMessage{}
+	_ NotificationsMessage = &BlockAnnounceHandshake{}
+)
+
```

Comment:
I don't understand why these vars are declared (without a name)?  How are they used?

Reply:
it's used to explicitly check that the message types satisfy the `NotificationsMessage` interface